upstream-examples/README.md

```diff
@@ -67,13 +70,11 @@ And then trigger a udev rule update:
 sudo udevadm trigger
 ```
 
-On the Raspberry Pi 5 the `raw-usb` interface has a [bug](https://bugs.launchpad.net/snapd/+bug/2071319) that does not provide the required permissions to the snap to access `raw-usb` devices.
-A workaround is to install the snap in [developer mode](https://snapcraft.io/docs/install-modes#heading--developer), which removes all confinement from the snap and disables automatic upgrades.
-This is insecure and not recommended, but currently the only way to use a Coral USB Accelerator with this snap on a Raspberry Pi 5.
-To install the snap in developer mode, do:
+On the Raspberry Pi 5 you also require the latest edge release of SnapD.
+To install it run this command:
 
 ```
-sudo snap install --dangerous ./tf-lite-examples_*+snap_amd64.snap --devmode
+sudo snap refresh snapd --edge
```

Comment:
Would be useful to mention which version is needed to let the user decide, or at least create an issue so we can track it.